packages/presentation/lib/screens/login/login_bloc.dart

```diff
@@ -75,6 +76,10 @@ class _LoginBloc extends BlocImpl<BaseArguments, LoginData>
   @override
   GlobalKey<FormState> get formStateGlobalKey => _formStateGlobalKey;
 
+  @override
+  set formStateGlobalKey(GlobalKey<FormState> value) =>
```

Comment:
create the key in bloc, no need to pass it from the widget I think

Reply:
Fixed.